chemprop/featurizers/atom.py

```diff
@@ -133,7 +136,117 @@ def num_only(self, a: Atom) -> np.ndarray:
         if a is None:
             return x
 
-        i = self.atomic_nums.get(a.GetAtomicNum() - 1, len(self.atomic_nums))
+        i = self.atomic_nums.get(a.GetAtomicNum(), len(self.atomic_nums))
         x[i] = 1
 
         return x
+
+    @classmethod
+    def default(cls):
+        """An implementation that includes features only for atoms in common molecules.
+        Includes all elements in the first four rows of the periodic table plus iodine. This is the default in Chemprop V2.
+        """
```

Comment:
```suggestion
    @classmethod
    def v2(cls):
        """An implementation that includes an atom type bit for all elements in the first four rows of the periodic table plus iodine.
        """
```

I feel like just naming this as `v2` is more idiomatic than `default` and then clarifying in the docstring what `default` actually means